eedl/image.py

```diff
@@ -166,18 +208,22 @@ def wait_for_images(self,
 
 			time.sleep(sleep_time)
 
-		if on_failure == "raise" and len(self.failed_tasks) > 0:
-			raise EEException(f"{len(self.failed_tasks)} images failed to export. Example error message from first"
-								f" failed image \"{self.failed_tasks[0].last_task_status['description']}\" was"
-								f" \"{self.failed_tasks[0].last_task_status['error_message']}\"."
-								f" Check https://code.earthengine.google.com/tasks in your web browser to see status and"
-								f" messages for all export tasks.")
+		if len(self.failed_tasks) > 0:
```

Comment:
Is there a reason you didn't write `if self.failed_tasks:`  instead?

Reply:
Yeah - I don't like the implicit type conversion there. That's fine, but this makes it clear that self.failed_tasks is a list and I'm checking whether it has items or not. I don't like it coercing values to boolean that aren't booleans. Is there a reason that the simpler form is better aside from being more readable, even if less explicit?